_posts/2016-01-27-continuous-integration.md

```diff
@@ -153,6 +153,6 @@ Other tests also get [a basic XML output file](https://github.com/bazelbuild/baz
 that contains only the result of the test (success or failure).
 
 To get your test results, you can also use the
-[Bazel dashboard](http://bazel.build/blog/2015/07/29/dashboard-dogfood.html),
+[Bazel dashboard](https://blog.bazel.build/2015/07/29/dashboard-dogfood.html),
```

Comment:
Probably better to use `{{ site.blog_site_url }}` but let's do that in a separate change.